cmd/cloud/organization.go

```diff
@@ -54,6 +62,37 @@ func newOrganizationSwitchCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func newOrganizationAuditLogs(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "audit-logs",
+		Aliases: []string{"al"},
+		Short:   "Manage your organization audit logs.",
+		Long:    "Manage your organization audit logs.",
+	}
+	cmd.AddCommand(
+		newOrganizationExportAuditLogs(out),
+	)
+	return cmd
+}
+
+func newOrganizationExportAuditLogs(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "export [organization_name] --output-file [output_file.json]",
```

Comment:
might want to make the organization name a flag to be consistent with other CLI commands, for example existing deployment variable command looks like: `astro deployment variable create FOO=BAR FOO2=BAR2 --deployment-id <deployment-id>`

Reply:
Sure. I also reorganized the flag so their scope is correct. Note organization name is mandatory.

![Screen Shot 2022-09-29 at 10 08 35 AM](https://user-images.githubusercontent.com/3437048/193095877-0a6e4aec-b2ea-4bd7-9622-9061d5290a16.png)